pkg/driver/driver_alicloud.go

```diff
@@ -292,12 +292,12 @@ func (c *AlicloudDriver) GetVolNames(specs []corev1.PersistentVolumeSpec) ([]str
 	names := []string{}
 	for i := range specs {
 		spec := &specs[i]
-		if spec.FlexVolume == nil || spec.FlexVolume.Options == nil {
-			// Not an aliCloud volume
-			continue
-		}
-		if name, ok := spec.FlexVolume.Options["volumeId"]; ok {
-			names = append(names, name)
+		if spec.FlexVolume != nil && spec.FlexVolume.Options != nil {
+			if name, ok := spec.FlexVolume.Options["volumeId"]; ok {
+				names = append(names, name)
+			}
+		} else if spec.CSI != nil && spec.CSI.VolumeHandle != "" {
```

Comment:
Should we also check if the field `spec.CSI.driver` matches the driver name(s) relevant to Alicloud?
https://kubernetes-csi.github.io/docs/drivers.html#production-drivers

Background: Typically other cloud providers such as AWS, Azure etc, have specific fields in the [`PersistentVolumeSource`](https://github.com/kubernetes/api/blob/master/core/v1/types.go#L177) struct, those fields were used in the corresponding drivers (eg. [AWS driver](https://github.com/gardener/machine-controller-manager/blob/rel-0.21.0/pkg/driver/driver_aws.go#L318)).

Reply:
By now, only  *diskplugin.csi.alibabacloud.com* will be considered.